### PR TITLE
[breaking] no longer allows :: fallback, forces namespace resolution to use $ syntax

### PR DESCRIPTION
**Pull request by gabrielcsapo**
*Why?*

This helps applications that are upgrading from component invocation to only use $ as they codemod over to angle brackets which this is required for.

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = function() {
     getChannelURL('canary')
   ]).then((urls) => {
     return {
+      useYarn: true,
```
Comment (rwjblue) [inline config/ember-try.js]: We shouldn't need to migrate to `yarn` just for this change. I don't mind using `yarn` in this repo, but this should be separate into a second PR focused on "migrating to yarn".

Comment (gabrielcsapo) [reply config/ember-try.js]: okay will open that PR.

**Comment by rwjblue** [comment]
For transparency:

* I just merged the changes that landed in v0.1.8 into master
* I'm going to land https://github.com/rwjblue/ember-holy-futuristic-template-namespacing-batman/pull/235
* release v0.2.0
* then we can land this and release as v1.0.0

**Comment by rwjblue** [comment]
@gabrielcsapo - Can you target `master` with these changes, and rebase?

**Pull request merged**